modules/gitops-release-notes-1-17-0.adoc

```diff
@@ -0,0 +1,149 @@
+
+[id="gitops-1-17-0-fixed-regular-for-repeated-flags-in-extraCommandArgs_{context}"]
+=== Fixed regular expression handling in `sourceNamespaces`
+
+* Before this update, incorrect regular expression processing in the `sourceNamespaces` field caused improper labeling of namespaces. As a result, the `argocd.argoproj.io/managed-by-cluster-argocd` label was not applied, affecting namespace management. With this update, Argo CD correctly processes regular expressions in `sourceNamespaces`. The {gitops-title} Operator correctly applies labels based on regex patterns, enabling management of multiple namespaces using a single expression. link:https://issues.redhat.com/browse/GITOPS-6675[GITOPS-6675]
+
+[id="gitops-1-17-0-extended-json-logging-support-for-additional-argo-cd-components_{context}"]
+=== Extended JSON logging support to additional Argo CD components
+
+* Before this update, setting `logFormat: json` in the Argo CD custom resource (CR) only enabled JSON logging for a limited set of components such as the API server, repo server, and application controller. With this update, setting `logFormat: json` is supported by the ApplicationSet controller and Notifications controller, allowing you to add logs in the JSON format. link:https://issues.redhat.com/browse/GITOPS-6668[GITOPS-6668]
++
+[NOTE]
+====
+Redis and Dex do not support configurable JSON logging and are unaffected by this change.
+====
+
+[id="gitops-1-17-0-fixed-UI-error-for-http-based-repository-credentials_{context}"]
+=== Fixed UI error when editing HTTP-based repository credentials
+
+* Before this update, the Argo CD Web UI failed to handle repositories hosted over HTTP due to URL validation errors. This prevented users from editing credentials for HTTP-based repositories through the Web UI. With this update, URL validation logic has been enhanced, and users can edit HTTP repository credentials directly in the Web UI. link:https://issues.redhat.com/browse/GITOPS-5969[GITOPS-5969]
+
+[id="gitops-1-17-0-authentication-tokens-redacted-in-argo-CD-e2e-test-logs_{context}"]
+=== Authentication tokens redacted in Argo CD E2E test logs
+ 
+* Before this update, Argo CD end-to-end (E2E) tests exposed authentication tokens in logs due to missing command argument redaction. This posed a security risk by potentially compromising authentication tokens during test execution. With this update, an authentication token redactor has been integrated into E2E test command execution. Authentication tokens are now hidden in logs, enhancing test security and protecting user credentials. link:https://issues.redhat.com/browse/GITOPS-3942[GITOPS-3942]
```

Comment:
AFAICT this can be removed, as it does not affect product code, only E2E tests (and our users/customers do not run E2E tests).